src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/Streaming/DataStreamReaderTests.cs

```diff
@@ -30,6 +31,12 @@ public void TestSignaturesV2_3_X()
 
             Assert.IsType<DataStreamReader>(dsr.Format("parquet"));
 
+            Assert.IsType<DataStreamReader>(
+                dsr.Schema(
+                    new StructType(new[]
```

Comment:
can we add an e2e test that tests `fromJson` behavior for all types similar to `TypesTests.cs` in `Microsoft.Spark.UnitTest`? You can do this as a follow up PR.

Reply:
Sure, will write a follow up PR to add an e2e test of `fromJson`.